modules/runtime/pages/bonita-docker-installation.adoc

```diff
@@ -331,10 +328,12 @@ Default value is *false*.
 If `ACCESSLOGS_FILES_ENABLED=true`, this optional environment variable allows to automatically delete access log files after a certain number of days. Default value is *30*.
 
 === HTTP_MAX_THREADS
-This optional environment variable allows to specify the maximum Http thread number Tomcat will use to serve HTTP/1.1 requests. Default value is *20*.
+This optional environment variable allows to specify the maximum Http thread number Tomcat will use to serve HTTP/1.1 requests. Directly modifies the *maxThreads* parameter of default connector in the *server.xml* file of the Tomcat inside the docker container.
+More information on the usefulness of this parameter can be found https://tomcat.apache.org/tomcat-8.5-doc/config/http.html[here]. Default value is *20*.
```

Comment:
At least point to version 9 of Tomcat doc (which is the version we use)

Reply:
ok